DESCRIPTION

```diff
@@ -30,6 +30,7 @@ BugReports: https://github.com/ropensci/vcr/issues
 Imports:
     cli,
     crul (>= 0.8.4),
+    curl,
```

Comment:
Already a hard dep of curl, so doesn't increase net dependencies.

Reply:
👍🏽